roles/fstat/tasks/main.yml

```diff
@@ -1,17 +1,41 @@
 ---
+- user:
+    name: "{{user}}"
+
 - name: Install required packages
   package:
     state: present
     name: "{{ item }}"
   with_items:
-  - python-virtualenv
-  - gcc
-  - git
+    - python-virtualenv
+    - gcc
+    - git
 
-# TODO create the directory /fstat, do the git checkout
-# automate update
-# set configuration file, and database creation
+# TODO set configuration file, and database creation
 # and selinux boolean
+
+- name: create fstat directory
+  file:
+    path: /fstat/code state=directory
+
+- name: Check if virtualenv exists
+  local_action: stat path=/fstat/env
```

Comment:
That is not correct, the local_action is gonna be used on the control system, not on the remote server.